source/code-snippets/usage-examples/find.js

```diff
@@ -0,0 +1,41 @@
+
+const { MongoClient } = require('mongodb');
+
+
+const client = new MongoClient(
+   'mongodb+srv://admin:admin@mflix-2sp0m.mongodb.net/test?retryWrites=true&w=majority',
+);
+
+async function run() {
+   try {
+      await client.connect();
+
+      const database = client.db('sample_mflix');
+      const collection = database.collection('movies');
+
+      // create a query document for a range match for a movie less than 15 minutes long
+      const query = { runtime: { $lt : 15 } };
+
+      // initialize an empty options object
+      options = {}
+
+      // sort our returned documents by ascending title
+      const sort = { title: 1 };
+
+      // returned documents should only contain title and the imdb object, no _id
+      const projection = { title : 1, _id: 0, imdb: 1 }
+      options.sort = sort;
+      options.projection = projection;
```

Comment:
I think we could condense this a bit without losing clarity if we combine all the definitions into the `options` declaration.
```js
const options = {
  // Sort the returned documents in ascending order by title
  sort: { title: 1 },
  // Include only the `title` and `imdb` fields in each returned document
  projection: { _id: 0, title: 1, imdb: 1 }
}
```